src/main/java/com/product/dao/CustomerDao.java

```diff
@@ -0,0 +1,13 @@
+package com.product.dao;
+
+import java.net.MalformedURLException;
+
+import org.springframework.http.ResponseEntity;
+
+import com.product.model.CustomerResponse;
+
+public interface CustomerDao {
+
+	public ResponseEntity<CustomerResponse> getCustomerResponseById(int id) throws MalformedURLException;
```

Comment:
Wait whaaaaaaat..... Dao Layer should not return ResponseEntity...

Reply:
Done.